cloud: add faq about how to import specified columns, and delete outdated files (#14117)
pingcap/docs: develop/dev-guide-insert-data.md

<CustomContent platform="tidb-cloud">

> - Data import: [Data Import Task](/tidb-cloud/import-sample-data.md) page in the TiDB Cloud console. You can import **Dumpling** exported data, a **CSV** file, or [Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md). It also supports reading data from a local disk, Amazon S3 cloud disk, or GCS cloud disk.

Author: ```suggestion
- Data import: [Data Import Task](/tidb-cloud/import-sample-data.md) page in the TiDB Cloud console. You can import **Dumpling** exported data, a **CSV** file, or [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md). It also supports reading data from a local disk, Amazon S3 cloud disk, or GCS cloud disk.
```
